app/src/main/java/com/eveningoutpost/dexdrip/Services/DexCollectionService.java

```diff
@@ -140,34 +160,70 @@ public void onSharedPreferenceChanged(SharedPreferences prefs, String key) {
         }
     };
 
+    public static byte getBridgeBattery() {
```

Comment:
I'm not happy about it either, and am happy to listen to recommendations. 

 
John Stevens
M - +61 418526349

> On 19 May 2015, at 23:59, Stephen Black <notifications@github.com> wrote:
> 
> In app/src/main/java/com/eveningoutpost/dexdrip/Services/DexCollectionService.java:
> 
> > @@ -140,34 +160,70 @@ public void onSharedPreferenceChanged(SharedPreferences prefs, String key) {
> >          }
> >      };
> >  
> > +    public static byte getBridgeBattery() {
> I feel like this is a strange way to get a value from the service, I would rather it be stored somewhere and then retrieved by the other class (either in the database or sharedprefs)
> 
> —
> Reply to this email directly or view it on GitHub.
>